vector/src/main/res/xml/vector_settings_notification_default.xml

```diff
@@ -6,22 +6,18 @@
 
         <im.vector.app.core.preference.VectorCheckboxPreference
             android:key="SETTINGS_PUSH_RULE_MESSAGES_IN_ONE_TO_ONE_PREFERENCE_KEY"
-            android:persistent="false"
```

Comment:
Are you sure you want to make the setting persistent? I think we get the value from the current push rule, so no need to store the values in a XML files.

Reply:
Ah yea, I was looking at the wrong android documentation for this attribute. Will put it back to false thanks.